main/src/main/java/org/mobilitydata/gtfsvalidator/table/GtfsCalendarDateSchema.java

```diff
@@ -29,10 +30,12 @@
 public interface GtfsCalendarDateSchema extends GtfsEntity {
   @FieldType(FieldTypeEnum.ID)
   @Required
+  @PrimaryKey
```

Comment:
Changes to this GtfsCalendarDateSchema and GtfsFrequencySchema make sense to me. However, they are adding extra validations and may potentially reject some feeds that are considered valid right now. Can we modify these files in a separate pull request?

Reply:
There is going to be a tricky chicken-and-egg problem here.  Depending on what mechanism we come up for translations.txt validation support, we need to come up with something that can gracefully handle having the changes to calendar date and frequency split into a separate PR.  Let's see how the other conversation goes before making this change.

Reply:
I am fine with changing GtfsCalendarDateSchema and GtfsFrequencySchema here if we don't have any feeds at Google that break this requirement (from thousands of feeds, there is usually at least one that breaks any assumption we make. Kind of Hyrum's Law for GTFS).

Reply:
I did a large-scale eval on production feeds and didn't find any new validation errors when run against the new @PrimaryKey additions.  I think this is good to go.

Reply:
Thanks for running the experiment!